### DISCUSSION TITLE
"or" operator using the --scope flag when task running

### DISCUSSION THREAD

**Discussion by seanblonien** [vercel/turborepo#483] [Help]
In Lerna, I used this syntax to build two non-dependent packages in parallel:
`lerna run build --sort --scope=@orgname/appname-{package1,package2}`
and simply changing to turbo didn't work
`turbo run build --scope=@orgname/appname-{package1,package2} --no-deps`

How do I do an `or` operation without just repeating the flag again and again?

Note, assume I have a `package3` with same org name, and same appname prefix which I don't want to run, thereby preventing me from just doing `--scope=@orgname/appname-*`

Repeating the flag does work, however, it is a pretty bad syntax when you consider all packages in a monorepo are commonly prepended with an org name, let alone other package prefixes
`turbo run build --scope=@orgname/appname-package1 --scope=@orgname/appname-package2 --no-deps`

**Comment by anthonyshew** [comment]
Marking as outdated since `--scope` flag is deprecated!
